### ISSUE TITLE
the validation set of WIDER_FACE not found

### ISSUE THREAD

**Issue by sukibean163** [chenjun2hao/CenterFace.pytorch#32]
baidu url https://pan.baidu.com/s/1b5Uku0Bb13Zk9mf7mkZ3FA fail

**Comment by lihua-hua** [comment]
https://pan.baidu.com/s/1wfENmkz-ID4RiP-gbZDWFA 提取码: m53d

**Comment by lihua-hua** [comment]
@sukibean163  https://pan.baidu.com/s/1wfENmkz-ID4RiP-gbZDWFA 提取码: m53d

**Comment by sukibean163** [comment]
@lihua-hua thanks, it works!
